### PR TITLE
fix: Load remote resources even if expressions in non requried parameters resolve

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: cypress/pages/workflow.ts [hunk 1]
```diff
@@ -126,12 +128,16 @@ export class WorkflowPage extends BasePage {
...
 			this.getters.nodeCreatorSearchBar().type(nodeDisplayName);
 			this.getters.nodeCreatorSearchBar().type('{enter}');
-			if (keepNdvOpen) return;
-			cy.get('body').type('{esc}');
+			if (opts?.action) {
+				nodeCreator.getters.getCreatorItem(opts.action).click();
+			}
+			else if (!opts?.keepNdvOpen) {
+				cy.get('body').type('{esc}');
+			}
```
Comment (OlegIvaniv) [inline cypress/pages/workflow.ts:140]: We could simplify this by calling the method below which already supports this behaviour:
```ts
addInitialNodeToCanvas: (nodeDisplayName: string, opts?: { keepNdvOpen?: boolean, action?: string }) =>
			this.getters.canvasPlusButton().click();

			this.actions.addNodeToCanvas(nodeDisplayName, false, opts?.keepNdvOpen, opts?.action);
		}
```

**Review by OlegIvaniv — CHANGES_REQUESTED** [review]
Works as expected, have just a few E2E suggestions

File: cypress/pages/workflow.ts (hunk 1 above)
Comment (mutdmour) [reply cypress/pages/workflow.ts:144]: tried this but some tests started failing.. maybe we can keep this as for now

**Review by OlegIvaniv — DISMISSED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.6.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.6.0)

**Pull request merged**